install_config/topics/using_gcp_load_balancer-proc.adoc

```diff
@@ -0,0 +1,113 @@
+////
+Module included in the following assemblies:
+
+install_config/configuring_gce.adoc
+////
+
+= Using the GCP external load balancer as a service
+
+{product-title} can leverage the GCP load balancer capacity by exposing services
+externally using a `LoadBalancer` service. {product-title} creates the load
+balancer in GCP and creates the proper firewall rules.
+
+[NOTE]
+====
+Currently, port 10256 must be opened manually for service load balancers
+to work in GCP. See https://bugzilla.redhat.com/show_bug.cgi?id=1583833 for
+more information.
+====
+
+.Procedure
+
+. Create a `LoadBalancer` service:
++
+[source,json]
+----
+{
+  "kind": "Service",
+  "apiVersion": "v1",
+  "metadata": {
+    "name": "myservice"
+  },
+  "spec": {
+    "ports": [{
+      "port": 9876,
+      "targetPort": 9876
+    }],
+    "selector": {
+      "app": "example"
+    },
+    "type": "LoadBalancer"
+  }
+}
+----
+
+. Create a new application:
++
+[source,bash]
+----
+$ oc new-app openshift/hello-openshift
+----
+
+. Expose the load balancer service:
++
+----
+$ oc expose dc hello-openshift --name='hello-openshift-external' --type='LoadBalancer'
+----
+
+. To verify that the service has been created:
++
+----
+$ oc get svc
+NAME                       TYPE           CLUSTER-IP       EXTERNAL-IP     PORT(S)                         AGE
+hello-openshift            ClusterIP      172.30.62.10     <none>          8080/TCP,8888/TCP               20m
+hello-openshift-external   LoadBalancer   172.30.147.214   35.230.97.224   8080:31521/TCP,8888:30843/TCP   19m
+----
++
+{product-title} creates the required objects in the GCP infrastructure:
+
+. Set the firewall rules:
++
+[source,bash]
+----
+$ gcloud compute firewall-rules list | grep k8s
+k8s-4612931a3a47c204-node-http-hc        my-net  INGRESS    1000      tcp:10256
+k8s-fw-a1a8afaa7762811e88c5242010af0000  my-net  INGRESS    1000      tcp:8080,tcp:8888
+----
++
+[NOTE]
+====
+The above firewall rules are applied to instances tagged with
+`<openshift_gcp_prefix>ocp`. For example, if the value of the
+`openshift_gcp_prefix` parameter is set to `mycluster`, you must tag the nodes
+with `myclusterocp`. See
+link:https://cloud.google.com/vpc/docs/add-remove-network-tags[Adding and
+Removing Network Tags] for more information on how to add network tags to
+Compute Engine VM instances.
+====
+
+. Set the health checks:
++
+[source,bash]
+----
+$ gcloud compute http-health-checks list | grep k8s
+k8s-4612931a3a47c204-node        10256  /healthz
+----
+
+. Load Balancer:
```

Comment:
I don't know what this step is doing. Any tips?

Reply:
So, when creating loadbalancer type services, some objects are created in GCP automatically by OCP such as:

* firewall rules
* health checks
* target pools
* load balancers
Those commands are to show users how to check and see the objects that are created.

Reply:
Oohh I see now.